package.json

```diff
@@ -96,7 +96,7 @@
     "i18n:js": "npx babel --config-file ./babel.makepot.js -o ./lang/tmp-babel.log assets --ignore \"**/*.test.js\",\"./assets/dist\",\"./assets/vendor\",\"./build\"",
     "i18n:php": "wp i18n make-pot --merge=lang/tmp-sensei-lms-js.pot --skip-js --exclude=build --headers='{\"Last-Translator\":null,\"Language-Team\":null,\"Report-Msgid-Bugs-To\":\"https://wordpress.org/support/plugin/sensei-lms\"}' . lang/sensei-lms.pot",
     "i18n:build": "npm run build:assets && npm run i18n:clean && npm run i18n:js && npm run i18n:php && node ./scripts/pot-dist-references.js && npm run i18n:clean",
-    "format-js": "wp-scripts format-js",
```

Comment:
It was changed in Gutenberg. More details: https://github.com/renatho/gutenberg/commit/06fd22e0f5922db0cd4ca1ae87b31887817a2f0b